lib/lan_connection.js

```diff
@@ -12,7 +12,8 @@ mdns.excludeInterface('0.0.0.0');
 function LANConnection(opts) {
   this.auth = {};
   this.ip = Array.isArray(opts.addresses) ? opts.addresses[0] : opts.host;
-  this.auth.host = opts.host || '';
+  this.host = opts.host || '';
+  this.auth.host = this.ip;
```

Comment:
whats the auth obj used for now that controller.js doesn't use connection.auth anymore?